staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1448,6 +1450,11 @@ type VirtualMachineStatus struct {
 	// +nullable
 	// +optional
 	MemoryDumpRequest *VirtualMachineMemoryDumpRequest `json:"memoryDumpRequest,omitempty" optional:"true"`
+
+	// InterfaceRequests indicates a list of interfaces added or removed from the VMI template and
+	// hotplug on an active running VMI.
+	// +listType=atomic
+	InterfaceRequests []VirtualMachineInterfaceRequest `json:"interfaceRequests,omitempty" optional:"true"`
```

Comment:
Could you add this in the change that actually uses it?
It is hard to understand now why it is needed.